index.js

```diff
@@ -144,7 +144,7 @@ class App extends MatrixPuppetBridgeBase {
       ghostIntent.client.sendReadReceipt (event);
       return this.receiptHistory.delete(roomId);      
     } catch (err) {
-      debug('could not send read event', err.message);
+      console.log('could not send read event', err.message);
```

Comment:
I actually don't remember why I made this change. I think it had been polluting the logs because `debug` was undefined, but this was quite a while back before last night when I came back around to look into the group message issue.

Reply:
Okay! Good point!